src/components/moviesDisplay/moviesDisplay.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+import { Navigate } from 'react-router-dom';
+
+import './moviesDisplay.css';
+
+
+const MoviesDisplay = ({ moviesList }) => {
```

Comment:
It was also okay if we fetched the movies directly in this moviesDisplay component.